src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -4195,7 +4195,7 @@ class MavlinkStreamDistanceSensor : public MavlinkStream
 			msg.min_distance = dist_sensor.min_distance * 100.0f; /* m to cm */
 			msg.max_distance = dist_sensor.max_distance * 100.0f; /* m to cm */
 			msg.current_distance = dist_sensor.current_distance * 100.0f; /* m to cm */
-			msg.covariance = dist_sensor.covariance;
+			msg.covariance = dist_sensor.variance * 1e4f; // m^2 to cm^2
```

Comment:
`1e-4f`

Reply:
To covert from uORB `distance_sensor`'s meters^2 to MAVLInk's cm^2, we should multiply by `1e4f`, don't we?

Reply:
Yeah nevermind. Sorry